doc/engineering-reference/src/surface-heat-balance-manager-processes/conduction-finite-difference-solution.tex

```diff
@@ -16,106 +42,174 @@ \subsection{Basic Finite Difference Solution Approach}\label{basic-finite-differ
                       k_E\frac{T_{i-1}^{j}-T_{i}^{j}}{\Delta x}\right)
 \label{eq:InternalNodeImplicitEquation}
 \end{equation}
-
-Where:
-
-T = node temperature
-
-Subscripts:
-
-i = node being modeled
-
-i+1 = adjacent node to interior of construction
-
-i-1 = adjacent node to exterior of construction
-
-j+1 = new time step
-
-j = previous time step
-
-$\Delta$t = calculation time step
-
-$\Delta$x = finite difference layer thickness (always less than construction layer thickness)
-
-C\(_{p}\) = specific heat of material
-
-k\(_{w}\) = thermal conductivity for interface between i node and i+1 node
-
-k\(_{E}\) = thermal conductivity for interface between i node and i-1 node
-
-$\rho$ = density of material
-
-Then, this equation is accompanied by a second equation that relates enthalpy and temperature.
+%
+where:
+\begin{wherelist}
+\item[T] node temperature
+\item[\Delta t] calculation time step
+\item[\Delta x] finite difference layer thickness (always less 
+than construction layer thickness)
+\item[C_p] specific heat of material
+\item[k_W] thermal conductivity for interface between node $i$ and node $i+1$
+\item[k_E] thermal conductivity for interface between node $i$ and node $i-1$
+\item[\rho] density of material
+\end{wherelist}
+%
+with superscipts and subscripts:
+%
+\begin{wherelist}
+\item[i] node being modeled
+\item[i+1] adjacent node to interior of construction
+\item[i-1] adjacent node to exterior of construction
+\item[j+1] new time step
+\item[j] previous time step
+\end{wherelist}
+%
+Then, this equation is accompanied by a second equation that relates enthalpy
+and temperature.
 
 \begin{equation}
-{h_i} = HTF\left( {{T_i}} \right)
+h_i = \text{HTF}\left(T_i\right)
 \end{equation}
-
+%
```

Comment:
Hmm.

Reply:
This prevents a paragraph break and the resulting indent.

Reply:
I think I might've known that at one point.  That's why my fuzzy memory search just returned "Hmm".  I would've expected `\noindent` but this is nice and clean.